### ISSUE TITLE
Is there a way to 'read' the trajectory information from the twix file?

### ISSUE THREAD

**Issue by mscipio** [pehses/twixtools#7]
I believe that `ismrmrd` has an option (when you read the acquisitions line by line) to also read the coordinate of the trajectory for each acquisition.

Is there a way to extract that same information using this tool?

**Comment by pehses** [comment]
Sorry for the late reply.

Unfortunately, in contrast to ismrmd, twix files do not explicitely include the trajectory information (there is no field in the protocol or the mini data header). In case of radial data, the line counter usually tells you the projection angle (even number of projections: pi * line / Nproj; odd: 2 * pi * line / Nproj). For ramp-sampling in EPIs, the shape of the gradient is stored in the protocol (see calc_regrid_traj() in recon_helpers.py). It is possible that non-cartesian sequences store at least some information about the trajectory in the same or similar fields as well ('alRegridMode' can be set to spiral).

So, in summary, it always depends on the sequence implementation on how to get information about the trajectory from the protocol, but there's unfortunately as far as I know no way to store the complete trajectory within the protocol of the twix file. I suspect that this may change in the future.
